src/components/BadDatePicker.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <v-date-picker
+        @input="handleUpdate"
+        :value="value"
+        :min="tomorrow()"
```

Comment:
I would suggest to create prop for the min value as well and to define the tomorrow() method in the BookingForm.vue component. Because in the future we might use the datepicker without min date or another min date.

Reply:
Yes, this makes sense to me! :)